msfextract

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env ruby
+# -*- coding: binary -*-
+
+class MsfExtractError < StandardError; end
+class HelpError < StandardError; end
+class UsageError < MsfExtractError; end
+
+require 'tmpdir'
+require 'optparse'
+require 'timeout'
+require 'open3'
+require 'nokogiri'
+require 'fileutils'
+
+def run_cmd(cmd)
```

Comment:
This really should be updated to take the binary and an array of arguments. That'll fix all of your issues where if one of the paths has a space in it, your tool will break. Also in the unlikely event that this is exposed remotely, it'll reduce the chances that there's something like a command execution vulnerability.